drivers/modem/Kconfig.ublox-sara-r4

```diff
@@ -77,4 +77,21 @@ config MODEM_UBLOX_SARA_R4_INIT_PRIORITY
 	  Note that the priority needs to be lower than the net stack
 	  so that it can start before the networking sub-system.
 
+config MODEM_UBLOX_SARA_R4_NETWORK_STATUS
+	bool "Enable support for network status indication"
+	help
+	  Choose this setting to use a modem GPIO pin as network indication.
+
+if MODEM_UBLOX_SARA_R4_NETWORK_STATUS
+
+config MODEM_UBLOX_SARA_R4_NETWORK_STATUS_PIN
+	string "Network status indication gpio_id"
+	default "16"
```

Comment:
Why this is a string and not an int?

Reply:
Woops.  I picked up this patch from @weinholtendian -- I will make this an int.